src/pybind/rados/rados.pyx

```diff
@@ -3192,6 +3194,43 @@ returned %d, but should return zero on success." % (self.name, ret))
         self.require_ioctx_open()
         return SnapIterator(self)
 
+    def get_id(self):
```

Comment:
i'd suggest use `get_pool_id()`. as id in this context could be ambiguous.

Reply:
done